packages/core/src/sass/app/_common/base/common.scss

```diff
@@ -141,6 +141,10 @@ html,
     }
 }
 
+.link.dark {
+    color: var(--brand-red-grey);
+}
+
```

Comment:
Is this being used? And should it follow the normal convention of `.link--dark` instead?

Reply:
Only for this case. Blank bolded links